README.md

```diff
@@ -58,6 +58,7 @@ Within the file `config.yml`:
 - Enter the executable name in the `engine: name` field (In Windows you may need to type a name with ".exe", like "lczero.exe")
 - If you want the engine to run in a different directory (e.g., if the engine needs to read or write files at a certain location), enter that directory in the `engine: working_dir` field.
   - If this field is blank or missing, the current directory will be used.
+  - IMPORTANT NOTE: If this field is used, the running engine will look for files and directories (Syzygy tablebases, for example) relative to this path, not the `engine: dir` field.
```

Comment:
I think that if `engine: working_dir` isn't set, the engine will look for files relative to `lichess-bot.py` and not `engine: dir`.

Reply:
This should also be changed in `config.yml.default`.